test/unit/modern.server.test.js

```diff
@@ -37,6 +34,31 @@ describe('modern server mode', () => {
     expect(response).toContain('arrow:function(){return"build test"}')
   })
 
+  test('should contain legacy http2 pushed resources', async () => {
+    const { headers: { link } } = await rp(url('/'), {
+      resolveWithFullResponse: true
+    })
+    expect(link).toEqual(
+      `</_nuxt/runtime.js>; rel=preload; as=script${
+        ''}, </_nuxt/commons.app.js>; rel=preload; as=script${
+        ''}, </_nuxt/app.js>; rel=preload; as=script${
+        ''}, </_nuxt/pages_index.js>; rel=preload; as=script`
+    )
+  })
+
+  test('should contain module http2 pushed resources', async () => {
+    const { headers: { link } } = await rp(url('/'), {
+      headers: { 'user-agent': modernUA },
+      resolveWithFullResponse: true
+    })
+    expect(link).toEqual(
+      `</_nuxt/modern-runtime.js>; rel=preload; as=script${
```

Comment:
Maybe use an array + join here instead of the (clever!) template literal hack?

Reply:
OK, I'll change to it.